### PR TITLE
Update help channels to point to formium org

### PR DESCRIPTION
**Pull request by jssee**
Currently, the URLs pointing to the repo/issues are not uniform or correct.
It's not a huge deal since it seems all redirects are current working correctly,
but consistency is nice

### CODE REVIEW

**Comment by jssee** [comment]
@agilgur5 good catch, updated!

File: src/messages.ts [hunk 1]
```diff
@@ -11,7 +11,6 @@ export const help = function() {
   return `
     Only ${chalk.green('<project-directory>')} is required.
     If you have any problems, do not hesitate to file an issue:
-      ${chalk.cyan('https://github.com/formik/tsdx/issues/new')}
```
Comment (agilgur5) [inline src/messages.ts:14]: think you might have accidentally made a mistake here as it got deleted instead of updated

Comment (jssee) [reply src/messages.ts:14]: sorry, this was an oversight. fixed now.

**Comment by jssee** [comment]
@agilgur5 i cant rebase and fix merge conflicts, i  think it might have to do with permissions?

**Comment by agilgur5** [comment]
@jssee ah so #765 introduced a conflict here.. Not sure what kind of error you're getting as command line `git` shouldn't normally give permissions errors. I also have no control over permissions in this repo in any case, so if that were actually the case, I wouldn't be able to resolve 😕

**Comment by jssee** [comment]
@agilgur5 no problem, i figured out what the conflict was, i was editing a page that no longer existed. I dropped that commit, hopefully this one is good to go

**Comment by agilgur5** [comment]
@allcontributors please add @jssee for code

**Pull request merged**